msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md

```diff
@@ -43,7 +43,9 @@ At a high level, the import process consists of the following:
 
 Since existing data is being migrated, maintaining the original message timestamps and preventing messaging activity during the migration process are key to recreating the user's existing message flow in Teams. This is achieved as follows:
 
-1. [Create a new team](/graph/api/team-post?view=graph-rest-beta&tabs=http) with a back-in-time timestamp using the team resource  `createdDateTime`  property.  
+1. [Create a new team](/graph/api/team-post?view=graph-rest-beta&tabs=http&preserve-view=true) with a back-in-time timestamp using the team resource  `createdDateTime`  property. 
+
+> **NOTE**:  The `createdDateTime` field will only be populated for instances of a team or channel that are migrated.
 
```

Comment:
1 and 2 are done in same request I am not sure making them 2 different steps in docs is worth it. It might confuse people.